src/hooks/useAnalyticsPageView.ts

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import {useAnalyticsContext} from '../contexts/useAnalyticsContext';
-import {UnknownRecord} from '../types/common';
 
-export function useAnalyticsPageView(params: UnknownRecord): void;
-export function useAnalyticsPageView(callback: () => UnknownRecord): void;
-export function useAnalyticsPageView(callback: () => Promise<UnknownRecord>): void;
-export function useAnalyticsPageView(paramsOrCallback: UnknownRecord | (() => Promise<UnknownRecord> | UnknownRecord)) {
+export function useAnalyticsPageView(params: string): void;
+export function useAnalyticsPageView(callback: () => string): void;
+export function useAnalyticsPageView(callback: () => Promise<string>): void;
+export function useAnalyticsPageView(paramsOrCallback: string | (() => Promise<string> | string)) {
   const analytics = useAnalyticsContext();
 
   const pageView = async () => {
     const params = typeof paramsOrCallback === 'function' ? await paramsOrCallback() : paramsOrCallback;
```

Comment:
```suggestion
export function useAnalyticsPageView(pathname: string): void;
export function useAnalyticsPageView(callback: () => string): void;
export function useAnalyticsPageView(callback: () => Promise<string>): void;
export function useAnalyticsPageView(pathnameOrCallback: string | (() => Promise<string> | string)) {
  const analytics = useAnalyticsContext();

  const pageView = async () => {
    const pathname = typeof pathnameOrCallback === 'function' ? await pathnameOrCallback() : pathnameOrCallback;
```

pathname으로 파라미터 네이밍 변경하면 좋을 것 같아요~!